deploy/070-CronJob.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           restartPolicy: OnFailure
           containers:
             - name: run
-              image: trustify-xtask
+              image: ghcr.io/trustification/xtask:latest
```

Comment:
That won't work right now, as the image must contain the AWS S3 client. Which this one does not. At least not today.

Reply:
Ah, I missed that ... I'll add it to the job, it seems more natural place for it

Reply:
Reverted back to using image stream for the moment.

Aws cli installation needs unzip package, which is not available in the xtask image.

We'll need to at least add unizp to the image. But also while I was adding aws installation to the job, it didn't sit right. I need to think a bit more where to put it.